### PR TITLE
Fix double down key bug

### LINKED ISSUE #348: [PE-D][Tester B] Cycling down in command history after cycling up requires 2 down arrow presses instead of just 1
## Expectation

When i press the up arrow key to access the last command, and I want to cycle back down to the newer command, I should only have to press the down arrow key once.

## Actual

After pressing up to cycle up, I have to press the down arrow key twice for the first command that I try to cycle down on.

## How to reproduce

Suppose I ran the commands: (but you can run any commands, as long as there are some in the history)
- `help`

1. Press the up arrow key once cycle up to go up to the last ran command `help`

2. Press the down arrow key once

I expect to see a empty command input, but it remains at `help`

- Press the down arrow key a second time

It successfully cycles to the empty command input.

This occurs as long as you press the down button after any number series of up presses.

-------------
Labels: `type.FunctionalityBug` `severity.VeryLow`
original: chuajunyu/ped#6

### PR DESCRIPTION
**Pull request by KennethTeo2002**
Fixes #348

### CODE REVIEW

**Review by joeltio — CHANGES_REQUESTED** [review]
This fixes the problem but introduces another bug:

1. Enter `list` then `view 1`
2. Up, Up, Down, Down, Up

Notice that the command becomes `list` instead of `view 1` now.

**Review by joeltio — APPROVED** [review]
LGTM

File: src/main/java/bizbook/ui/CommandHistory.java [hunk 1]
```diff
@@ -47,8 +48,12 @@ public String getPreviousCommand() {
      * @return the next command, or an empty string if at the end of the history
      */
     public String getNextCommand() {
-        if (currentIndex >= 0 && currentIndex < history.size()) {
-            return history.get(currentIndex++);
+        assert currentIndex >= -1;
```
Comment (sheenkerr) [inline src/main/java/bizbook/ui/CommandHistory.java:51]: Maybe can add `-1` as a magic literal, since it used in the initialization and here?

**Review by sheenkerr — APPROVED** [review]
Aside from the potential adding of a magic literal, LGTM

**Pull request merged**